packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -7119,15 +7110,22 @@ export default class Meeting extends StatelessWebexPlugin {
     }
 
     let floorRequestNeeded = false;
+    if (tracks.screenShare?.video || (this.isMultistream && tracks.screenShare?.audio)) {
+      // @ts-ignore
+      this.webex.internal.newMetrics.submitClientEvent({
```

Comment:
in theory someone can call publishTracks() twice, first time just with the video share track, second time with audio share track - in that case we will send 'client.share.initiate' twice and we probably want to sent it just once. I guess we just want to send it when floorRequestNeeded is true